harness/determined/common/experimental/workspace.py

```diff
@@ -57,6 +57,28 @@ def reload(self) -> None:
 
         self._hydrate(workspace_bindings)
 
+    def resource_pools(self) -> Optional[List[str]]:
```

Comment:
nit: because I expect this returns a list no matter if there are bound resource pools or not, I think the best return type is just `List[str]`.